pkg/streamingpromql/types/data.go

```diff
@@ -47,9 +47,9 @@ func (i *InstantVectorSeriesDataIterator) Reset(data InstantVectorSeriesData) {
 // It returns the next point with the lowest timestamp.
 // If h is not nil, the value is a histogram, otherwise it is a float.
 // If no more values exist ok is false.
-func (i *InstantVectorSeriesDataIterator) Next() (t int64, f float64, h *histogram.FloatHistogram, ok bool) {
+func (i *InstantVectorSeriesDataIterator) Next() (t int64, f float64, h *histogram.FloatHistogram, hIndex int, ok bool) {
```

Comment:
We should describe the hIndex in the docstring

Reply:
Done: d6eaad2